src/Alpha_complex/include/gudhi/Alpha_complex.h

```diff
@@ -146,6 +146,10 @@ class Alpha_complex {
   std::unique_ptr<Triangulation> triangulation_;
   /** \brief Kernel for triangulation_ functions access.*/
   A_kernel_d kernel_;
+  /** \brief Vertices to be inserted first by the create_complex method to avoid quadratic complexity.
+   * It isn't just [0, n) if some points have multiplicity (only one copy appears in the complex).
+  */
+  std::vector<std::size_t> vertices_;
```

Comment:
Yes, we don't know here what is a `Vertex_handle`.
In CGAL triangulation, I can only find these 2 types (cf. [Triangulation_ds_iterators](https://github.com/CGAL/cgal/blob/4e0a2522f3f99c4cb3ec1980434a48d1319ffb6b/Triangulation/include/CGAL/Triangulation/internal/Triangulation_ds_iterators.h#L33-L34)). And `difference_type` doesn't seem to be used for vertices. Am I wrong ?

Reply:
Not in CGAL, in Gudhi! In Alpha_complex.h, line 110, you used `ptrdiff_t`, for the same purpose of storing a Vertex_handle without knowing the type Vertex_handle. What I am suggesting is that however we choose this type, it would make sense to use the same in both places. If that's size_t, then you should change ptrdiff_t to size_t. Better yet, create a Fake_vertex_handle typedef (ideally we would change the code so we know what Vertex_handle is, but that may require changing the API, so it will wait).

Reply:
Use Internal_vertex_handle type which is a ptrdiff_t for all on [11dec62](https://github.com/GUDHI/gudhi-devel/pull/713/commits/11dec62e555e446fb34a1f5ba3d0c9aba1a7e956)